tests/test_utils/test_tensor_move.py

```diff
@@ -1,42 +1,83 @@
 import pytest
 
 from colossalai.utils.cuda import get_current_device
-from colossalai.utils.memory_utils.utils import colo_model_data_tensor_move, colo_model_data_tensor_move_inline
+from colossalai.utils.memory_utils.utils import colo_tensor_mem_usage, colo_model_data_tensor_move, colo_model_data_tensor_move_inline, colo_set_process_memory_fraction, colo_cuda_memory_capacity, colo_model_data_move_to_cpu, colo_model_tensor_clone
 from colossalai.utils import free_port
-from colossalai.zero.sharded_param import ShardedTensor
+from colossalai.zero.sharded_param.tensorful_state import StatefulTensor
 import colossalai
 
 import torch
 
 from functools import partial
 import torch.multiprocessing as mp
 
+def _run_colo_tensor_mem_usage():
+    for i in range(1):
+        if i == 1:
+            t1 = StatefulTensor(torch.randn(2,2))
+            t2 = StatefulTensor(torch.randn(4,4))
+            c1 , g1 = colo_tensor_mem_usage(t1)
+            c2 , g2 = colo_tensor_mem_usage(t2)
+            assert c1*4 == c2
+            assert g1*4 == g2
+        else:
+            t1 = torch.randn(2,2)
+            t2 = torch.randn(4,4)
+            c1 , g1 = colo_tensor_mem_usage(t1)
+            c2 , g2 = colo_tensor_mem_usage(t2)
+            assert c1*4 == c2
+            assert g1*4 == g2
 
-def _run_colo_model_data_tensor_move_inline():
-    for t in [torch.randn(2, 3), ShardedTensor(torch.randn(2, 3))]:
-        colo_model_data_tensor_move_inline(t, torch.device(f"cuda:{get_current_device()}"))
-        assert t.device == torch.device(f"cuda:{get_current_device()}")
+def _run_colo_set_process_memory_fraction_and_colo_cuda_memory_capacity():
+    frac1 = colo_cuda_memory_capacity()
+    colo_set_process_memory_fraction(0.5)
+    frac2 = colo_cuda_memory_capacity()
+    assert frac2*2 == frac1
 
+def _run_colo_model_data_tensor_move_inline():
+    for t in [StatefulTensor(torch.randn(2,3)), torch.randn(2,3)]:
+        colo_model_data_tensor_move_inline(t, torch.device("cuda:3"))
+        assert t.device == torch.device("cuda:3")
```

Comment:
The assert pass only when the env has 4 GPU.

see
@pytest.mark.parametrize("world_size", [1, 4])
we may run with 1 GPU.

Reply:
change it to current_device